File: BeerViewer/Models/kcsapi/kcsapi_quest.cs
﻿namespace BeerViewer.Models.kcsapi
{
	public class kcsapi_quest
	{
		public int api_no { get; set; }
		public int api_category { get; set; }
		public int api_type { get; set; }
		public int api_state { get; set; }
		public string api_title { get; set; }
		public string api_detail { get; set; }
		public int[] api_get_material { get; set; }
		public int api_bonus_flag { get; set; }
		public int api_progress_flag { get; set; }
	}
}

File: WindowFrame/src/KanColle/Raw/kcsapi_quest.ts
// tslint:disable:class-name

export interface kcsapi_quest {
	api_no: number;
	api_category: number;
	api_type: number;
	api_state: number;
	api_title: string;
	api_detail: string;
	api_get_material: number[];
	api_bonus_flag: number;
	api_progress_flag: number;
}
export interface kcsapi_questlist {
	api_count: number;
	api_page_count: number;
	api_disp_page: number;
	api_list: kcsapi_quest[];
	api_exec_count: number;
}
export interface kcsapi_req_quest_requests {
	api_quest_id: number;
}
